sapi/cli/php_cli_server.c

```diff
@@ -171,6 +171,9 @@ typedef struct php_cli_server_client {
 	char *current_header_name;
 	size_t current_header_name_len;
 	unsigned int current_header_name_allocated:1;
+	char *current_header_value;
+	size_t current_header_value_len;
+	enum { NONE=0, FIELD, VALUE } last_header_element;
```

Comment:
I'd suggest prefixing the enum members, e.g. `HEADER_NONE, HEADER_FIELD, HEADER_VALUE`.